.github/workflows/test-samples.yaml

```diff
@@ -56,21 +56,25 @@ jobs:
                         --env="OPENEO_AUTH_PROVIDER_ID=CDSE" \
                         -d --rm registry.cloudferro.com/vito/jupyterhub:rc-89266`
           echo Started container ${container_id}
-          echo Waiting for jupyterlab to start ... 
+          echo Waiting for jupyterlab to start ...
           sleep 60
           echo "container_id=${container_id}" >> "$GITHUB_ENV"
 
+      - name: Check code style with Black
+        run: |
+          black --check .
```

Comment:
this will now fail because `black` is not available.

In general I don't think it's necessary to have black (or other linters) in the `test-samples.yaml` action. It might be clearer to have a separate github action for this